app/helpers.php

```diff
@@ -117,12 +117,17 @@ function human_diff_string(\Carbon\Carbon $ts1, \Carbon\Carbon $ts2, $absolute =
         return 'unknown length';
     }
 
+    if ($ts2 <= \Carbon\Carbon::minValue() || $ts2 >= \Carbon\Carbon::maxValue()
+        || $ts1 <= \Carbon\Carbon::minValue() || $ts1 >= \Carbon\Carbon::maxValue()) {
+        return 'unreliable dates';
```

Comment:
This check should throw an exception rather than returning a string - this will mean any issues can be fixed further up the stack, rather than pretending there's not an issue.